person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rental
   attr_accessor :name, :age
```

Comment:
- [ ] As required for the project, kindly ensure that `Getters` and `Setters` for `@rentals` instance variables are present. Using `getter` and `setter` methods allows to access and modify the value of a variable, provides a controlled and predictable interface for other objects to interact with the variable, while still maintaining the principles of encapsulation and data hiding. i.e. ``attr_accessor :name, :age, :rentals``. Also remember to correct the typo error, you wrote `:rental` instead of `:rentals`